### PR TITLE
Lint comments on exported objects (functions, types, ...)

### LINKED ISSUE #267: Linter does not report undocumented publics
### Location
`.golangci.yml`

### Problem
The [default suppression](https://golangci-lint.run/usage/configuration/) filter of golangci-lint removes the "should have comment or be unexported" warnings.  
Therefore public types/functions without comment currently pass the CI unnoticed.  

### Proposal
This can be disabled with `exclude-use-default: false` but would then require us to copy the remaining defaults that we still want to use.  
Just copying some of the default values does not seem to work so well either, since some are deprecated…

### PR DESCRIPTION
**Pull request by matthiasgeihs**
Closes #267

### CODE REVIEW

**Comment by matthiasgeihs** [comment]
Close and reopen to trigger DCO.

File: backend/sim/wallet/address.go [hunk 1]
```diff
@@ -63,13 +62,13 @@ func (a *Address) Bytes() []byte {
 
 // ByteArray converts an address into a 64-byte array. The returned array
```
Comment (manoranjith) [inline backend/sim/wallet/address.go:63]: byteArray....

Comment (matthiasgeihs) [reply backend/sim/wallet/address.go]: Not directly related to my changes, but shows that the linter only cares about exported functions...

**Comment by matthiasgeihs** [comment]
Addressed your comment.

**Review by manoranjith — APPROVED** [review]
LGTM.

**Pull request merged**